README.md

```diff
@@ -10,6 +10,30 @@
 
 BigchainDB is a blockchain database.
 
+## Run and test BigchainDB from the `master` branch
+Running and testing the latest version of BigchainDB is easy. Make sure you have a recent version of [Docker Compose installed](https://docs.docker.com/compose/install/) in your host.
+
+Whenever you are ready, fire up a terminal and run:
+```
+git clone https://github.com/bigchaindb/bigchaindb.git
+cd bigchaindb
+make run
+```
+
+To know the IP of the instance running, execute in a new terminal `make ip`.
+
+There are also other commands you can execute:
+- `make start`: Run BigchainDB from source and damonize it (stop it with `make stop`).
```

Comment:
daemonize ?

Reply:
Weirdly enough, [yes](https://en.wikipedia.org/wiki/Daemon_(computing)).

Reply:
Ah, can you add an 'e' then? :D

Reply:
damonizee?

Reply:
we can replace it with an evil emoji!